docs/form_hooks/response_hooks.rst

```diff
@@ -1,3 +1,72 @@
 Response hooks
 ==========================================================
 
+.. js:method:: onResponse()
+
+    Called prior to the response form the Form submission is processed that handles downloading files, displaying a success message, and/or redirecting to another URL.
```

Comment:
```suggestion
    Called prior to the response form, the Form submission is processed, which handles downloading files, displaying a success message, and/or redirecting to another URL.
```

Reply:
I think this is correct grammar, wasn't 100% sure on the context for the sentence.

Reply:
That `form` is supposed to be `from` but the whole sentence is janky. I’ll fix it.